src/components/step-flow/step-flow.test.tsx

```diff
@@ -1,279 +1,333 @@
 import React from "react";
-import { render, RenderResult, screen } from "@testing-library/react";
+import { render, screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import enGB from "../../locales/en-gb";
 import { StepFlow } from "./index";
 import { StepFlowHandle, Steps } from "./step-flow.component";
 import Button from "../button";
+import I18nProvider from "../i18n-provider";
 
-describe("Step Flow component", () => {
-  function generateLimitedVariations(): [Steps, Steps][] {
-    const variations: [Steps, Steps][] = [];
+function generateLimitedVariations(): [Steps, Steps][] {
+  const variations: [Steps, Steps][] = [];
 
-    for (let totalSteps = 1; totalSteps <= 8; totalSteps++) {
-      for (let currentStep = 1; currentStep <= totalSteps; currentStep++) {
-        variations.push([totalSteps as Steps, currentStep as Steps]);
-      }
+  for (let totalSteps = 1; totalSteps <= 8; totalSteps++) {
+    for (let currentStep = 1; currentStep <= totalSteps; currentStep++) {
+      variations.push([totalSteps as Steps, currentStep as Steps]);
     }
-
-    return variations;
   }
 
-  function generateCurrentStepOverTotalStepsVariations(): [Steps, Steps][] {
-    const variations: [Steps, Steps][] = [];
+  return variations;
+}
 
-    for (let totalSteps = 1; totalSteps <= 8; totalSteps++) {
-      for (let currentStep = totalSteps + 1; currentStep <= 8; currentStep++) {
-        variations.push([totalSteps as Steps, currentStep as Steps]);
-      }
-    }
+function generateCurrentStepOverTotalStepsVariations(): [Steps, Steps][] {
+  const variations: [Steps, Steps][] = [];
 
-    return variations;
+  for (let totalSteps = 1; totalSteps <= 8; totalSteps++) {
+    for (let currentStep = totalSteps + 1; currentStep <= 8; currentStep++) {
+      variations.push([totalSteps as Steps, currentStep as Steps]);
+    }
   }
 
-  function calculateStepStateIndexes(
-    totalSteps: number,
-    currentStepParam: number
-  ) {
-    let currentStep = currentStepParam;
+  return variations;
+}
 
-    if (currentStep > totalSteps) {
-      currentStep = totalSteps;
-    }
-
-    const stepsBefore = currentStep - 1;
-    const stepsAfter = totalSteps - currentStep;
+function calculateStepStateIndexes(
+  totalSteps: number,
+  currentStepParam: number
+) {
+  let currentStep = currentStepParam;
 
-    return [stepsBefore, stepsAfter];
+  if (currentStep > totalSteps) {
+    currentStep = totalSteps;
   }
 
+  const stepsBefore = currentStep - 1;
+  const stepsAfter = totalSteps - currentStep;
+
+  return [stepsBefore, stepsAfter];
+}
+
+describe("Step Flow component", () => {
```

Comment:
suggestion(non-blocking): Describe isn't required

I know you've just moved it and we will revisit this file but it was flagging as a change so thought I'd mention. Not the end of the world for now as long as we tidy it up later